Classes/Foundation/HTTPCookie.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 
 public extension HTTPCookie {
-    /// Stringify an instance of a HTTPCookie            
+    /// Stringify an instance of a HTTPCookie in JavaScript notation
```

Comment:
"Stringify an instance of a HTTPCookie into a string of key/value pairs, e.g. "id=1234; domain=de; path=foo; ..."